src/resolvers/resolver.cr

```diff
@@ -44,7 +44,7 @@ module Shards
       end
     end
 
-    protected def install_path
+    def install_path
```

Comment:
I don't agree. If you are creating something public you have to guarantee it is correct and the way to do that is automated tests.

But it is up to you.